Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityToolkit.SDK.Input.asmdef

```diff
@@ -1,6 +1,7 @@
 {
     "name": "Microsoft.MixedReality.Toolkit.SDK.Input",
```

Comment:
Daft question, but do we really need to subdivide the SDK project, other than editor code?
This seems overkill as the SDK layer is a "Take it or Leave it" type of project.

Reply:
This is mainly to ensure that the assemblies are built in the right order.

Reply:
The custom assembly definitions give us clear dependencies, and complication order.

Reply:
Not for the SDK, it's not needed.  It's meant to be the simplest of projects, just drop in and use.   Doesn't need the same level of complexity as the main project.

Reply:
If you'd like to break the out later to make it work be my guest. 

But you'll likely run into compile issues

Reply:
Done